extensions/positron-python/src/client/positron/createEnvApi.ts

```diff
@@ -0,0 +1,74 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// --- Start Positron ---
+// eslint-disable-next-line import/no-unresolved
+import * as positron from 'positron';
+// --- End Positron ---
```

Comment:
Don't think we need this anymore

```suggestion
// eslint-disable-next-line import/no-unresolved
import * as positron from 'positron';
```